app/src/main/java/woowacourse/paint/custom/view/CanvasView.kt

```diff
@@ -0,0 +1,69 @@
+package woowacourse.paint.custom.view
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.graphics.Canvas
+import android.graphics.Paint
+import android.graphics.Path
+import android.util.AttributeSet
+import android.view.MotionEvent
+import android.view.View
+import com.now.domain.BrushWidth
+import woowacourse.paint.custom.model.CurveLine
+import woowacourse.paint.custom.model.CurveLines
+import woowacourse.paint.presentation.uimodel.BrushColorUiModel
+
+class CanvasView(
+    context: Context,
+    attributeSet: AttributeSet,
+) : View(context, attributeSet) {
+
+    private val curveLines = CurveLines()
+    private var curveLine = CurveLine(Path(), Paint())
+
+    init {
+        isFocusable = true
+        isFocusableInTouchMode = true
+    }
+
+    override fun onDraw(canvas: Canvas) {
+        super.onDraw(canvas)
+        curveLines.draw(canvas)
+    }
+
+    @SuppressLint("ClickableViewAccessibility")
+    override fun onTouchEvent(event: MotionEvent): Boolean {
+        val pointX = event.x
+        val pointY = event.y
+
+        when (event.action) {
+            MotionEvent.ACTION_DOWN -> {
+                startDrawing(pointX, pointY)
+                invalidate()
+            }
+            MotionEvent.ACTION_MOVE -> {
+                keepDrawing(pointX, pointY)
+                invalidate()
+            }
+            else -> super.onTouchEvent(event)
+        }
+        return true
```

Comment:
```suggestion
        invalidate()
        return true
```
아주 사소한 거긴 한데요..
중복되는 코드를 이렇게 빼내올 수 있을 것 같네요 ~